.eslintrc.js

```diff
@@ -28,123 +32,50 @@ module.exports = {
     node: true,
     jest: true,
   },
+  reportUnusedDisableDirectives: true,
   overrides: [
     {
       files: ['**/*.ts', '**/*.tsx'],
-      extends: ['airbnb-typescript', 'plugin:unicorn/recommended', 'prettier'],
+      extends: [
+        'airbnb-typescript',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/strict',
+        // TODO: Opt-in to a stricter ruleset in the future
+        // 'plugin:@typescript-eslint/strict-type-checked',
+        'plugin:@typescript-eslint/stylistic-type-checked',
+      ],
```

Comment:
`@typescript-eslint` was set up incorrectly unfortunately and didn't do anything. Now its in effect and caught a lot of lint issues.